test/e2e/network/ingress.go

```diff
@@ -1141,6 +1142,14 @@ var _ = SIGDescribe("Ingress API", func() {
 		framework.ExpectNoError(err)
 		framework.ExpectEqual(updatedStatus.Status.LoadBalancer, statusToUpdate.Status.LoadBalancer, fmt.Sprintf("updated object expected to have updated loadbalancer status %#v, got %#v", statusToUpdate.Status.LoadBalancer, updatedStatus.Status.LoadBalancer))
 
+		ginkgo.By("get /status")
+		ingResource := schema.GroupVersionResource{Group: "networking.k8s.io", Version: ingVersion, Resource: "ingresses"}
+		gottenStatus, err := f.DynamicClient.Resource(ingResource).Namespace(ns).Get(context.TODO(), createdIngress.Name, metav1.GetOptions{}, "status")
```

Comment:
Did you find that the DynamicClient was the only way to use the Namespace call?

Reply:
Using the DynamicClient this way the first option that worked for me. Would you prefer it done another way?

Reply:
is the Get not working for this?
https://github.com/kubernetes/kubernetes/blob/30b0ebd6d43de3b9eb6b78d877612a4d7aed85a8/test/e2e/network/ingress.go#L1063

Reply:
that does not get the status subresource